packages/paste-website/src/components/site-wrapper/SiteHeader.tsx

```diff
@@ -13,6 +13,19 @@ import {SiteBodyContext} from './SiteBodyContext';
 export const SiteHeader: React.FC = () => {
   const {isPasteTheme} = React.useContext(SiteBodyContext);
   const theme = useTheme();
+  const [value, setValue] = React.useState('');
+
+  React.useEffect(() => {
+    if (typeof window !== `undefined` && window.location != null) {
+      // @ts-ignore
```

Comment:
Ignoring because `docsearch` comes from the external js. Error is:
`Property 'docsearch' does not exist on type 'Window & typeof globalThis`

Reply:
You can add the check for `window.docsearch` to the condition above, should remove the ts error